analysis/metrics.py

```diff
@@ -98,12 +73,16 @@ class AggregateErrorMetrics:
 
```

Comment:
Please extend the docstring with ephasizing that this cross-partition aggregate metrics